File: Java/palindrome_number9/Solution.java
class Solution {
    public boolean isPalindrome(int x) {
    int a = 0;
    if(x < 0 || x != 0 && x % 10 == 0)
    {
        return false;
    }

    while( x> a)
    {
        a = a * 10 + x % 10;
        x /= 10;
    }
    return(x == a || x == a/10);
    }
}

File: C/palindrome_number9/Solution.c
bool isPalindrome(int x) {
    if(x<0 || x!=0 && x%10==0)
    return false;

    int a=0;
    while(x>a)
    {
        a = a*10+x%10;
        x/=10;
    }
    return(x==a || x==a/10);
}
